charter/README.md

```diff
@@ -102,6 +102,16 @@ Chairs have responsibilities to their parent Working Group: they ensure that the
 
 Chairs have responsibility to all participants in transparently reporting the group's activity.
 
+Chairs also have the following regular duties within their Working Groups:
+* Collect agenda items prior to each meeting to ensure a smooth and focused Working Group meeting.
+* Ensure that the Working Group is notified of the agenda for the meeting in a timely manner advance of the meeting, and that if there is no agenda, the meeting is altered or cancelled accordingly.
```

Comment:
Note that persistent cancelling of meetings should be seen as a sign the working group may need restructuring, reorganization, etc.